MAUI/TreeView/Item-height-customization.md

```diff
@@ -0,0 +1,134 @@
+---
+layout: post
+title: Item Height Customization in .NET MAUI TreeView | Syncfusion
+description: Learn here all about Item Height Customization support in Syncfusion .NET MAUI TreeView (SfTreeView) control and more.
+platform: .NET MAUI
+control: SfTreeView
+documentation: ug
+---
+
+# Item Height Customization in .NET MAUI TreeView (SfTreeView)
+
+The TreeView provides various options to customize the height of items. To achieve this customization, please walkthrough the below sections:
+
+## Customize Item Height
+
+The TreeView allows customizing the height of items by setting the `ItemHeight` property. The default value of this property is `40d`. This property can be customized at runtime.

```

Comment:
The TreeView allows for customization of item height by setting the `ItemHeight` property. The default value of this property is `40d`. This property can be customized at runtime.